docs/Design.md

```diff
@@ -44,29 +44,31 @@ from the controller is an outgoing message it sends it to the CAN bus.
 The ```EventConsumerService``` may react to consumed events by calling a user registered callback so that
 the user sketch can act on this event for example to turn on an LED or move a servo.
 
-The user sketch may produce events when are passed on the ```EventProducerService``` object which in turn passes
-this event as a Command via the Controller to the transport object.
+The user sketch may produce events that are managed by the ```EventProducerService``` object which then passes
+the event as a Command via the Controller to the transport object.
 
 ### Dataflow
-Most of VLCB functionality use a message object ```VlcbMessage``` that is used to pass incoming and
+Most of the VLCB functionality uses a message object ```VlcbMessage``` that passes incoming and
 outgoing messages around via the Command bus. 
 The ```VlcbMessage``` object contains 8 bytes where the first is the op-code and the remaining 7 bytes
 are any optional data bytes for that op-code.
 
-The ```CanTransport``` object translates the VlcbMessage object to from an object that represents
-a CAN frame which contains an id, 8 bytes of data (same as the VlcbMessage) and the flags
+The ```CanTransport``` object translates the VlcbMessage object to form an object that represents
```

Comment:
I actually meant "to and from".

Reply:
OK.  We can do it like that and then remove the new line 66.  Do you want me to amend?

Reply:
I've changed to reflect your original intention.